src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Machine exc1 = new Excavator();
+        Machine bull1 = new Bulldozer();
+        Machine truck1 = new Truck();

```

Comment:
please reconsider variable names - shortened names are not a good option
and try not to use numbers in variable names

also I would suggest creating an array and iterating with `for-each` loop